packages/teleport/src/components/DatabaseList/DatabaseList.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Column,
+  SortHeaderCell,
+  Cell,
+  renderLabelCell,
+  TextCell,
+  SortTypes,
+} from 'design/DataTable';
+import Table from 'design/DataTable/Paged';
+import isMatch from 'design/utils/match';
+import { sortBy } from 'lodash';
+import { Database } from 'teleport/services/database';
+
+function DatabaseList(props: Props) {
+  const { databases = [], searchValue = '', pageSize = 100 } = props;
+
+  const [sortDir, setSortDir] = React.useState<Record<string, string>>({
+    name: SortTypes.DESC,
+  });
+
+  function sortAndFilter(search) {
+    const filtered = databases.filter(obj =>
+      isMatch(obj, search, {
+        searchableProps: ['name', 'desc', 'protocol', 'type', 'uri', 'tags'],
+        cb: searchAndFilterCb,
+      })
+    );
+
+    const columnKey = Object.getOwnPropertyNames(sortDir)[0];
+    const sorted = sortBy(filtered, columnKey);
+    if (sortDir[columnKey] === SortTypes.ASC) {
+      return sorted.reverse();
+    }
+
+    return sorted;
+  }
+
+  function onSortChange(columnKey: string, sortDir: string) {
+    setSortDir({ [columnKey]: sortDir });
+  }
+
+  const data = sortAndFilter(searchValue);
+
+  return (
+    <div>
```

Comment:
the `StyledTable` doesn't need to be wrapped with anything b/c it has no siblings. (i know in NodesList it does, not sure why, could be left over code that we forgot to clean up after making changes).

Reply:
You're right but it'll need a wrapper now that it has `<SearchInput />` as a sibling